src/components/v5/common/Filter/partials/FilterOptions.module.css

```diff
@@ -0,0 +1,7 @@
+.li {
+  @apply flex min-h-[2.188rem] items-center gap-3 transition-all duration-normal text-md text-gray-900 mr-2 hover:bg-gray-50 hover:text-gray-900 hover:font-medium cursor-pointer rounded p-2 [&>i]:ml-3;
```

Comment:
2.188rem is 35,008px
Why do you use this value?

Reply:
This color is set globally text-gray-900
Are you sure do you need it here?

Reply:
It's better to add <span> container to the icon and add this style: 'block ml-3' instead of use nested styles